Modules/Libs/QuestieLib.lua

```diff
@@ -665,7 +665,7 @@ function QuestieLib.GetSpawnDistance(spawnA, spawnB)
 end
 
 ---@param quest Quest
----@return number -- Questie.ICON_TYPE_X
+---@return number iconType The number representing the type of icon
```

Comment:
What is `iconType` used for? Does this bring some extra information?

Reply:
Yes, same as with the `added` in another file, names the return value.